metadata/metadata_service/entity/resource_type.py

```diff
@@ -5,6 +5,9 @@
 
 
 class ResourceType(Enum):
+    """
+    DEPRECATED, Please use amundsen_common/entity/resource_type.py instead
```

Comment:
Could we import it and do `ResourceType = amundsen_common.entity.ResourceType` instead?